conda_forge_tick/new_update_versions.py

```diff
@@ -119,6 +107,91 @@ def new_update_upstream_versions(
             node_count += 1
 
 
+def _update_upstream_versions_process_pool(
+    gx: nx.DiGraph, sources: Iterable[AbstractSource],
+) -> None:
+    futures = {}
+    # this has to be threads because the url hashing code uses a Pipe which
+    # cannot be spawned from a process
+    with executor(kind="dask", max_workers=20) as pool:
```

Comment:
Change the `max_workers` to 10 for lesser the burden of CircleCI
```suggestion
    with executor(kind="dask", max_workers=10) as pool:
```